### DISCUSSION TITLE
Status bars not updating...

### DISCUSSION THREAD

**Discussion by TruffleFries54** [dylandoamaral/uptime-card#160] [Q&A]
Howdy,

I’ve been working with HA for several months and am hooked!  I’m running Home Assistant in a Docker on a Unraid server and have been having problems with the update-card where it does not reflect the state over time.   That is, bar is gray and does not update with changes in state.   

I want to monitor a system (NUC) via ICMP and am using the [Home Assistant Ping (ICMP)](https://www.home-assistant.io/integrations/ping/) integration.  I followed the instructions from [How To Monitor Servers in Home Assistant ](https://smarthomepursuits.com/how-to-monitor-devices-in-home-assistant/?expand_article=1), loaded the update-card via HACS. 
  
The binary sensor for PING is functioning properly.  

```
binary_sensor:
  - platform: ping
    host: 192.168.3.71
    name: NUC
```

The card is reading the binary sensor.

 
![image](https://github.com/dylandoamaral/uptime-card/assets/112016291/69f39b41-2fda-4790-85c6-9e0c1865c4f7)

![image](https://github.com/dylandoamaral/uptime-card/assets/112016291/afb051d4-ba85-4697-8244-12f3b2014ed4)

 

Here’s the code for the update-card:

```
type: custom:uptime-card
entity: binary_sensor.nuc
icon: mdi:network
ko_icon: mdi:network-off
name: NUC
hours_to_show: 1
status_adaptive_color: true
color: {}
show:
  footer: false
alias: {}
bar: {}
update_interval: 30
```

I’ve read the [github](https://github.com/dylandoamaral/uptime-card/tree/main) documentation and other forums, but have not been able to identify a solution to my problem…   
Any suggestions are appreciated!

**Comment by dylandoamaral** [comment]
Maybe entity is case sensitive and should be binary_sensor.NUC. Did you manage to fix the issue or did you see the sensor in tgghe developper panel of home assistant ?
